<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BootstrapPage</class>
 <widget class="QWizardPage" name="BootstrapPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Bootstrap Parameters</string>
  </property>
  <property name="subTitle">
   <string>Bootstrap parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string># bootstrap replicates:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="replicates_spinBox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Plot Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="plot_path_le">
       <property name="text">
        <string>./r_tmp/bootstrap.png</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Plot Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="plot_title_le">
       <property name="text">
        <string>Bootstrap Histogram</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>horizontal label:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xlab_le">
       <property name="text">
        <string>Effect Size</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>131</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
